src/app/store.js

```diff
@@ -5,6 +5,7 @@ import IncidentListReducer from '../components/IncidentList/IncidentListSlice'
 export const store = configureStore({
   reducer: {
     declareIncidentForm: DeclareIncidentFormReducer,
-    ongoingIncidents: IncidentListReducer
+    ongoingIncidents: IncidentListReducer,
+
```

Comment:
I think we'll end up adding a searchResults property in this reducer. It'll end up being an array with all the contact objects that fit the search query. We just need to figure out how the reducer will work on the DatabaseMenu component.